### PR TITLE
Polymorphism in Document Form

### PR DESCRIPTION
**Pull request by Pranshu321**
<hr>

## Description 📜
I completed my issue work , i raised issue on the topic "polymorphism" on date 9/19/2021.
In this document or readme file i deeply talked about the types of polymorphism and with complete code snippets with understandable comments in simple language.
Topics covered

1. Compile time polymorphism and their types.
2. Run time polymorphism and their types.
And types of these polymorphism.
Issue number closed : #835 
<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)
 

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by gunjan-g** [comment]
Great work @Pranshu321 
Just put your file in the correct folder, Introduction to OOPs

**Review by gunjan-g — CHANGES_REQUESTED** [review]
Great work @Pranshu321
Just put your file in the correct folder, Polymorphism

File: C_CPP/polymorphism.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Polymorphism
```
Comment (Suvraneel) [inline C_CPP/polymorphism.md:1]: @Pranshu321 Put this file inside polymorphism folder

**Review by Suvraneel — CHANGES_REQUESTED** [review]

File: C_CPP/polymorphism.md (hunk 1 above)
Comment (Suvraneel) [reply C_CPP/polymorphism.md:1]: And edit that directories readme (like index to a book)

Comment (Pranshu321) [reply C_CPP/polymorphism.md:1]: ### Sir , i have done all the changes that you proposed please look at that once @Suvraneel 
1. Updation of readme file : #1906 
2.  Putting my file in classes & object folder : #1902

**Pull request closed without merging**